### PR TITLE
Release changelog for 2.8.9

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by kyrofa** [comment]
Isn't this supposed to be 2.9?

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+  [ Jamie Bennett ]
+  * Documentation: Use plugs instead of caps. (#507)
+
+  [ Chris Wayne ]
+  * Add in bash completion. (#453) (LP: #1570506)
+
+  [ Kyle Fazzari ]
+  * Make pull and build steps dirty if target arch changes. (#450)
+    (LP: #1564192)
+  * Add support for confinement property. (#501) (LP: #1580819)
```
Comment (kyrofa) [inline debian/changelog]: The epoch commit is missing, here.

**Comment by kyrofa** [comment]
This is also missing commit 6fabd20 (the schema change for plugs).

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+  [ Leo Arias ]
+  * autopkgtests: run the install examples tests in classic. (#481)
+    (LP: #1572764)
+
+  [ Matteo Bertini ]
+  * Fix typo in description: python2 -> python3. (#504)
+
+  [ Jamie Bennett ]
+  * Documentation: Use plugs instead of caps. (#507)
```
Comment (kyrofa) [inline debian/changelog]: I'm not sure this (or the one directly above) are particularly useful in the changelog.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,25 @@
+snapcraft (2.8.9) xenial; urgency=medium
+
+  [ Leo Arias ]
+  * autopkgtests: run the install examples tests in classic. (#481)
+    (LP: #1572764)
+
+  [ Matteo Bertini ]
+  * Fix typo in description: python2 -> python3. (#504)
```
Comment (sergiusens) [inline debian/changelog]: description of the python3 example maybe?

Comment (sergiusens) [inline debian/changelog]: oh, no LP bug for this, jus remove this entry

Comment (sergiusens) [inline debian/changelog]: or maybe not, but explain it better

**Comment by sergiusens** [comment]
closing in favor of #517

**Pull request closed without merging**